torch/ao/quantization/_pt2e/prepare.py

```diff
@@ -29,11 +29,11 @@ def _maybe_insert_input_and_output_observers_for_node(
     else:
         output_is_a_tensor = this_node_dtype_info is not None
 
-    skip_inserting_observers = (
-        not output_is_a_tensor
+    skip_inserting_input_observers = (
```

Comment:
nit: this should probably be `skip_inserting_input_and_output_observers`

Reply:
Changed and thanks.